### ISSUE TITLE
Ubuntu FireFox issue

### ISSUE THREAD

**Issue by Raiksler** [robertjanetzko/LegendsBrowser2#8]
Doesn't work for me on Ubuntu 22.04 in FireFox. Browser just don't start trough launcher, and there is never ending loading when i try to manual start trough FireFox.

**Comment by robertjanetzko** [comment]
can you try version 2.0.2? It should stop the infinite redirecting and show the underlying reason that caused the problem on the terminal. Can you send me the error message?

**Comment by ViceroyFaust** [comment]
I just tried to run the program in version 2.0.2 on Arch Linux with Firefox. It successfully opens the tab, however, the tab shows an error.
```
The page isn’t redirecting properly

Firefox has detected that the server is redirecting the request for this address in a way that will never complete.

    This problem can sometimes be caused by disabling or refusing to accept cookies.
```
Perhaps this can be of help? The original Java-based LegendsBrowser 1.19.2 works.

**Comment by mkalinski** [comment]
I think I might have triggered something similar when I moved the directory that LB2 remembered as "last". Have you tried removing the `~/.legendsbrowser/` directory?

**Comment by Siinamon** [comment]
On Gentoo, I am having the same problem with Vivaldi on a fresh run (no `~/.legendsbrowser/`) and cleared browser cache.

**Comment by jllllll** [comment]
This seems to be an issue on all OS. After looking at Process Monitor, it seems that the program gets stuck in a loop of trying to open whatever legends export was opened last and not stopping when it can't find it. Deleting the config file mentioned above fixes the issue.

**Comment by robertjanetzko** [comment]
> How do I find the `~/.legendsbrowser/` directory?
~ is your user directory. to display hidden files and folders press shift + cmd + .

**Comment by FrankIsAtank** [comment]
The same issue occurs on Windows 10 with Firefox and Edge, and on 

On Firefox, after a few seconds, I get : 
```
The page isn’t redirecting properly

Firefox has detected that the server is redirecting the request for this address in a way that will never complete.

    This problem can sometimes be caused by disabling or refusing to accept cookies.
```

On Edge I immediately get :

```
This page isn’t working right now localhost redirected you too many times.
To fix this issue, try [clearing your cookies](https://go.microsoft.com/fwlink/?linkid=2112473).
ERR_TOO_MANY_REDIRECTS
```

I even tried with Lynx and got a "Location URL is not absolute." error in loop, redirecting to `http://[ip address]:58881/load` until it gives up. I doubt this is a browser configuration error, it seems like a problem with the header.

It is also probably unrelated to cookies (as the other browsers suggest) since Lynx asks you to accept each cookie individually by default, and it did not propose any such cookie.

**Comment by robertjanetzko** [comment]
I finally got around to fix this issue. You can find the fix in verison 2.0.3 in the releases section
